android/src/main/kotlin/be/tramckrijte/workmanager/Extractor.kt

```diff
@@ -269,4 +280,64 @@ object Extractor {
                 }
                 .build()
     }
+
+    private fun extractInputData(call: MethodCall): Map<String, Any>? {
```

Comment:
Do we really need to _parse_ this map to a Kotlin Map on Android side?
Do we really care?
Can't we just store as an arbitrary `String` payload, but still return it as a `Map<String, dynamic>` on Dart side?

If so this method can be a whole lot simpler.